stubs/Php81.phpstub

```diff
@@ -13,6 +13,8 @@ namespace {
 
     interface BackedEnum extends UnitEnum
     {
+        /** @var non-empty-string $name */
+        public readonly string $name;
```

Comment:
For some reason, it was not picking up this property from `UnitEnum`

Reply:
I bet the reason is that interfaces in PHP cannot have properties.